src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -574,6 +663,12 @@ class GPT2DoubleHeadsModelOutput(ModelOutput):
             - 0 for tokens that are **masked**.
 
             `What are attention masks? <../glossary.html#attention-mask>`__
+        attention_block_mask (:obj:`torch.BoolTensor` of shape or broadcastable to :obj:`(batch_size, num_heads, sequence_length, sequence_length)`, `optional`):
```

Comment:
Maybe add a sentence that says "if None -> then `attention_block_mask` defaults to the `causal mask`